tests/fixtures/scenario-usegalaxy-dev.yml

```diff
@@ -4,10 +4,10 @@ global:
 tools:
   default:
     cores: 1
-    mem: cores * 3 # note, some clusters will tolerate more than this
+    mem: cores * 3 if cores else 1  # note, some clusters will tolerate more than this
     env: {}
     params:
-      nativeSpecification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={mem*1024}"
+      nativeSpecification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={mem*1024 if mem else 0}"
```

Comment:
would ‘else 1024’ make more sense here?

Reply:
Yes, it probably would. A more realistic example might omit the mem clause altogether if mem is not specified, but I didn't want to complicate the example too much.

We would really need to run the linter against usegalaxy-au (which I did btw). There are a number of errors flagged but curious to hear your thoughts on them. I think all of the errors were reasonable from a type checking point of view, but it does require us to make some modifications.